calculator_test.go

```diff
@@ -14,11 +15,182 @@ func TestAdd(t *testing.T) {
 	}
 }
 
-// func TestSubtract(t *testing.T) {
-// 	t.Parallel()
-// 	var want float64 = 2
-// 	got := calculator.Subtract(4, 2)
-// 	if want != got {
-// 		t.Errorf("want %f, got %f", want, got)
-// 	}
-// }
+
+func almostEqual(a, b float64) bool {
```

Comment:
Yes, this is a very good idea! Hopefully it's not necessary for our test cases, since they should be _exactly_ equal, but it's good practice in general to compare floating-point value this way.

Reply:
Yes probably not necessary in our minimal example but would like to keep it here future references